packages/node-cli/src/services/FileSystemService.js

```diff
@@ -110,6 +110,25 @@ module.exports = class FileSystemService {
 		}
 	}
 
+	emptyFolderRecursive(folderPath, exceptionPath) {
```

Comment:
I will name the second parameter, excludePath.

Reply:
Done!